explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -168,33 +223,11 @@ object SpectroscopyModesTable extends ItcColumn {
         .setMinWidth(50)
         .setMaxWidth(150),
       column(TimeColumnId, itc.forRow(cw, sn, _))
-        .setCell { c =>
-          val content: TagMod = c.value match {
-            case Left(nel)                        =>
-              if (nel.exists(_ == ItcQueryProblems.UnsupportedMode))
-                Popup(content = "Mode not supported", trigger = Icons.Ban.color("red"))
-              else {
-                val content = nel.collect {
-                  case ItcQueryProblems.MissingSignalToNoise => "Set S/N"
-                  case ItcQueryProblems.MissingWavelength    => "Set Wavelength"
-                  case ItcQueryProblems.GenericError(e)      => e
-                }
-                Popup(content = content.mkString_(", "), trigger = Icons.TriangleSolid)
-              }
-            case Right(ItcResult.Result(t, c))    =>
-              val secs = t.toMillis / 1000.0 * c
-              f"$secs%.0f s"
-            case Right(ItcResult.Pending)         =>
-              Icons.Spinner.spin(true)
-            case Right(ItcResult.SourceTooBright) =>
-              Popup(content = "Source too bright", trigger = Icons.SunBright.color("yellow"))
-          }
-          <.div(ExploreStyles.ITCCell, content)
-        }
+        .setCell(c => Reusable.implicitly(c.value).map(p => itcCell(p)))
```

Comment:
After a 2nd look at this, I don't think `itcCell` is being reused here.

The render function passed to `setCell` goes straight to `react-table` in JS land. So, it seems the `Reusable` is being recreated in each render here, without it getting compared to the previous instance.

Of course, there's a chance I may be wrong. Do you notice an improvement over just having `.setCell(c => itcCell(c.value))` ?

Reply:
I'm not sure either. I guess there will be nobody to handle the reusabillity result?

Reply:
Exactly, so the reusable value is just being resolved locally. In that case, we could be degrading performance instead of improving it.